server/entity/MediaRequest.ts

```diff
@@ -151,14 +152,15 @@ export class MediaRequest {
             omission: '…',
           }),
           image: `https://image.tmdb.org/t/p/w600_and_h900_bestv2${movie.poster_path}`,
-          media,
```

Comment:
why are we no longer attaching the media object here?

Reply:
Since we're passing the request object, it isn't really necessary.

Reply:
Actually, nevermind, that doesn't work, adding the media objects back lol.